test/units/cli/test_galaxy.py

```diff
@@ -444,3 +450,168 @@ def test_template_ignore_similar_folder(self):
 
     def test_skeleton_option(self):
         self.assertEquals(self.role_skeleton_path, context.CLIARGS['role_skeleton'], msg='Skeleton path was not parsed properly from the command line')
+
+
+class ValidCollectionTests(object):
+
+    expected_collection_dirs = ('docs', 'plugins', 'roles')
+
+    @classmethod
+    def setUpCollection(cls, collection_name, galaxy_args=None, skeleton_path=None):
```

Comment:
I'd turn this into a pytest fixture

Reply:
Yea I originally was following the existing test setup but looking back it's probably best to just ignore it and follow the pytest fixture standard.